src/main/kotlin/ch/wisv/choice/document/controller/DocumentController.kt

```diff
@@ -4,11 +4,13 @@ import ch.wisv.choice.document.model.Document
 import ch.wisv.choice.document.model.DocumentDTO
 import ch.wisv.choice.document.service.DocumentService
 import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.MediaType
 import org.springframework.web.bind.annotation.*
 import org.springframework.web.multipart.MultipartFile
 
 @RestController
-@RequestMapping("/document")
+@RequestMapping("/api/v1/document")
```

Comment:
Why is /api/v1 added?

Reply:
Because of the dashboard, so the context-path is just /choice

Reply:
Would we want the dashboard in a seperate repository? Why not?